src/Storage/Storage.Management/help/Start-AzStorageBlobCopy.md

```diff
@@ -29,7 +29,7 @@ Start-AzStorageBlobCopy [-SrcBlob] <String> -SrcContainer <String> -DestContaine
 Start-AzStorageBlobCopy -CloudBlob <CloudBlob> [-BlobBaseClient <BlobBaseClient>] -DestContainer <String>
  [-DestBlob <String>] [-PremiumPageBlobTier <PremiumPageBlobTier>] [-StandardBlobTier <String>]
  [-RehydratePriority <RehydratePriority>] [-Tag <Hashtable>] [-Context <IStorageContext>]
- [-DestContext <IStorageContext>] [-DestTagCondition <String>] [-TagCondition <String>] [-Force]
+ [-DestContext <IStorageContext>] [-DestTagCondition <String>] [-TagCondition <String>] [-Force] [-AsJob]
```

Comment:
https://github.com/Azure/azure-powershell/blob/b7e188390754a0a5c4f4323912e0b86374f8a691/src/Storage/Storage/Blob/Cmdlet/StartAzureStorageBlobCopy.cs#L306
I'm confused by -AsJob here, could you explain further?

Reply:
The cmdlet really support -asjob before,  https://github.com/Azure/azure-powershell/blob/919880fa25aa35539f6308bbec224e4d9a8e0484/src/Storage/Storage/Blob/Cmdlet/StartAzureStorageBlobCopy.cs#L306

But it looks the help is not updated. So when update help, the cmdlet is added to help.

Reply:
My point is seems like we commented AsJob in this command

Reply:
I think we should leave -Asjob in the cmdlet, and update help to make it aligned.

Reply:
@BethanyZhou 

You are right. I have removed the -asjob change from start-AzStorageBlobCopy help.
It's interesting why this parameter is generated when I update help last time, but it's removed when I update the help again.

Thanks for the catching!